### PR TITLE
fix(cirriculum): Clarify instructions in step 61 of cat photo app project

### LINKED ISSUE #48871: Clarify instructions and hints in step 61 of the cat photo app
### Describe the Issue

For step 61 of the cat photo app <https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-61>, since a lot of people don't read the instruction extra carefully to detect that the checked attribute is requested in 2 places, can we make it more obvious by writing the instruction as follows: Make both the first radio button and the first checkbox selected by default. (That is add the word “both” to the instruction)

In addition the hints can be clarified further by adding the id names to them as per @naomi-lgbt ‘s suggestion (via discord discussion)
For eg. Instead of “ The first radio button is missing the `checked` attribute.” use “ The first radio button (#indoor) is missing the `checked` attribute.”

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-html-by-building-a-cat-photo-app/step-61

### Your code

```

```

### Expected behavior

Update the instruction and hints to be more direct as per the suggestions above.

### Screenshots

_No response_

### System

N/A

### Additional context

File relative path: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804e5.md

### PR DESCRIPTION
**Pull request by sam-rice**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes [#48871](https://github.com/freeCodeCamp/freeCodeCamp/issues/48871)

Replaces the line of instruction referenced in the [issue](https://github.com/freeCodeCamp/freeCodeCamp/issues/48871) with @JordanMooree 's [suggestion](https://github.com/freeCodeCamp/freeCodeCamp/pull/48888#discussion_r1059857683).

### CODE REVIEW

File: curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804e5.md [hunk 1]
```diff
@@ -22,7 +22,7 @@ assert(
 );
 ```
 
-The first radio button is missing the `checked` attribute.
+Your first radio button with the `id` set to indoor should have the `checked` attribute.
```
Comment (Sboonny) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-html-by-building-a-cat-photo-app/5ef9b03c81a63668521804e5.md:25]: Translator may translate `indoor`, if it isn't in code block.

```suggestion
Your first radio button, with the `id` set to `indoor`, should have the `checked` attribute.
```

**Review by Sboonny — APPROVED** [review]
LGTM 🎉, thank you for your contribution to the page! 👍.

**Comment by sam-rice** [comment]
Hey @Sboonny,
Does the failed Chrome test just need to be re-run or is there an actual issue with the recent commit? The failed test doesn't seem connected to the recent change.

**Comment by Sboonny** [comment]
Your change doesn't affect that file, it was being flakey

**Review by JordanMooree — COMMENTED** [review]
LGTM 👍

**Review by JordanMooree — APPROVED** [review]

**Pull request merged**